web/src/engine/websites/ArgosScan.ts

```diff
@@ -55,7 +55,7 @@ export default class extends DecoratableMangaScraper {
         const vars = { id: id };
         const request = new FetchRequest(new URL('/graphql', this.URI).href);
         const operationName = 'project';
-        const data = await FetchGraphQL<ApiSingleResult<APIManga>>(request, operationName, gql, JSON.stringify(vars));
+        const data = await FetchGraphQL<ApiSingleResult<APIManga>>(request, operationName, gql, vars);
```

Comment:
Shouldn't `vars` still be a JSON serialised string?
The implementation in `FetchGraphQL` was not updated to support this type change.

Reply:
We end up getting the same result. Turns out serializing it was just because i wanted to pass it as a string . I tested all affected website and they works. And GraphQL is sensible about data types. Gotta test them again just to be sure

Reply:
I see, so embedding a stringified JSON object within another stringified JSON object was not by intention, but a workaround.

Reply:
Its working like a charm :)